tests/components/SearchBar.stories.tsx

```diff
@@ -55,3 +55,16 @@ DropdownExpanded.play = ({ canvasElement }) => {
   userEvent.keyboard('{enter}');
   userEvent.click(canvas.getByRole('textbox'));
 };
+
+export const HighlightDropdown = Primary.bind({});
+HighlightDropdown.play = ({ canvasElement }) => {
+  const canvas = within(canvasElement);
```

Comment:
oh does this make the searchbar do a mini "movie" when the page loads? that's kind of cool if it does it's like a mini acceptance test

Reply:
yeah, it goes through the steps really fast so you don't see the animation, but they have an experimental step-by-step debugging [feature](https://github.com/storybookjs/storybook/blob/next/addons/interactions/README.md) that we could turn on that would let you go through the different interactions individually